### PR TITLE
Provide documentation for Kubernetes events

### PR DESCRIPTION
**Pull request by ahmelsayed**
Signed-off-by: Ahmed ElSayed <ahmels@microsoft.com>

Closes https://github.com/kedacore/keda/issues/530

### CODE REVIEW

**Review by tomkerkhove — CHANGES_REQUESTED** [review]
Thanks for these docs!

I've added a remark on https://github.com/kedacore/keda/pull/1523#issuecomment-770180510 so I'll wait with reviewing until we decided on that.

Would you mind moving these docs from v2.0 to v2.2, please, since that's the version where this will be introduced.

File: content/docs/2.0/operate/_index.md [hunk 1]
```diff
@@ -7,4 +7,5 @@ weight = 1
 We provide guidance & requirements around various areas to operate KEDA:
 
 - Cluster ([link](./cluster))
-- Integrate with Prometheus ([link](./prometheus))
\ No newline at end of file
+- Integrate with Prometheus ([link](./prometheus))
+- Kubernetes Events ([link](./events))
```
Comment (tomkerkhove) [inline content/docs/2.0/operate/_index.md:11]: Can you move this to v2.2 please?

File: content/docs/2.2/operate/events.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
++++
+title = "Events"
+description = "Kubernetes Events emitted by KEDA"
+weight = 100
++++
+
+## Kubernetes Events emitted by KEDA
+
+KEDA emits the following [Kubernetes Events](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#event-v1-core)
```
Comment (tomkerkhove) [inline content/docs/2.2/operate/events.md:9]:
```suggestion
KEDA emits the following [Kubernetes Events](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#event-v1-core):

```

**Review by tomkerkhove — COMMENTED** [review]
This might be a stretch, but do you have raw payloads for these events as examples?

File: content/docs/2.0/operate/_index.md (hunk 1 above)
Comment (ahmelsayed) [reply content/docs/2.0/operate/_index.md]: oops, sorry forgot about this one.

**Review by tomkerkhove — APPROVED** [review]
LGTM, thanks a ton @ahmelsayed!

**Pull request merged**